requests/census_household_gb_eng.json

```diff
@@ -325,7 +332,7 @@
             "method": "POST",
             "url": "/questionnaire/household/{person_1_list_id}/confirm-dob/",
             "data": {
-                "confirm-date-of-birth-answer": "Yes",
+                "confirm-date-of-birth-answer": "Yes, I am {age_in_years} years old",
```

Comment:
Each of these placeholders in the data should really be replaced with actual values. As it is, it's a bit confusing as to whether this is expected or not.